ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -81,22 +82,23 @@ public void getSize(
     Uri uri = Uri.parse(uriString);
     ImageRequest request = ImageRequestBuilder.newBuilderWithSource(uri).build();
 
-    DataSource<CloseableReference<CloseableImage>> dataSource =
-      Fresco.getImagePipeline().fetchDecodedImage(request, mCallerContext);
+    DataSource<CloseableReference<PooledByteBuffer>> dataSource =
+      Fresco.getImagePipeline().fetchEncodedImage(request, mCallerContext);
```

Comment:
Fresco only supports limited source types for encoded image:
https://github.com/facebook/fresco/blob/v1.3.0/imagepipeline/src/main/java/com/facebook/imagepipeline/core/ProducerSequenceFactory.java#L169

For example: `SOURCE_TYPE_LOCAL_CONTENT` (content://) which is returned from `RollCamera` is not supported

Use `ContentResolver.openInputStream` together with  [BitmapUtil.decodeDimensions(...)](https://github.com/facebook/fresco/blob/df2854a76786f58e044a71617309cc3c0a0c358c/imagepipeline-base/src/main/java/com/facebook/imageutils/BitmapUtil.java#L91) for those not supported types.

Actually `BitmapUtil.decodeDimensions` probably has better performance than loading the fully encoded image since it only needs to decode the metadata. So I suggest use that when possible.